core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -438,6 +440,11 @@ public DefaultFileSystemMaster(BlockMaster blockMaster, CoreMasterContext master
     mSyncManager = new ActiveSyncManager(mMountTable, this);
     mTimeSeriesStore = new TimeSeriesStore();
     mAccessTimeUpdater = new AccessTimeUpdater(this, mInodeTree, masterContext.getJournalSystem());
+
+    // Sync executors should allow core threads to time out
```

Comment:
What does this do? (and what is the behavior with and without this change?)

Reply:
If no metadata syncs are occurring the executor will spin down threads to decrease resource utilization

Reply:
The impact is that the first time a sync needs to occur there will be a slight overhead for that RPC to create the threads. The overhead is equivalent to the time to create the threads needed to process the sync.